tests/server/test_fork_handler.py

```
@@ -0,0 +1,354 @@
+import pytest
+
+from nerve.core.nodes.llm import OpenRouterNode, StatefulLLMNode
+from nerve.core.nodes.llm.chat import Message
+from nerve.server.engine import build_nerve_engine
+from nerve.server.protocols import Command, CommandType
+
+
+class MockEventSink:
+    """Mock event sink for testing."""
+
+    def __init__(self):
```

Ruff (0.14.10)

Missing return type annotation for special method `__init__`

Add return type annotation: `None`

(ANN204)